cscs-checks/apps/namd/namd_check.py

```diff
@@ -135,3 +127,10 @@ def setup_perf_vars(self):
                     'eiger:mc': {'days_ns': (0.05, None, 0.05, 'days/ns')},
                     'pilatus:mc': {'days_ns': (0.05, None, 0.05, 'days/ns')}
                 }
+
+    @performance_function('days/ns')
```

Comment:
In the future we should consider inverting this metric to `ns/day`, which is more standard.